.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: node_js
+matrix:
+  include:
+    - node_js: "6"
+    - node_js: "7"
+dist: trusty
+cache:
+  yarn: true
+  directories:
+    - node_modules
+install:
```

Comment:
Looks like it works with this solution 
https://github.com/paulirish/pwmetrics/pull/29